workflow/scripts/ts_simulators.py

```diff
@@ -267,11 +270,12 @@ class recombination_rate(BaseSimulator):
         "pop_size": 1e4,
         # RANDOM PARAMETERS (UNIFORM)
         "recombination_rate": [0, 1e-8],
+        "dummy": [0, 1e-8],
```

Comment:
what's up with this dummy random param?

Reply:
that's me debugging; i'm not done checking stuff out